backend/src/main/java/de/bund/digitalservice/ris/caselaw/adapter/transformer/FieldOfLawTransformer.java

```diff
@@ -43,22 +43,21 @@ public static FieldOfLaw transformToDomain(
       builder.linkedFields(linkedFields);
     }
 
+    if (fieldOfLawDTO.getChildren() != null) {
+      builder.hasChildren(!fieldOfLawDTO.getChildren().isEmpty());
```

Comment:
[Non-blocking]
Can't we just say builder.hasChildren(true); like in line 57? is there a difference to set it differently?